analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -296,7 +296,8 @@ def construct_config_handler(cls, args, context):
             host_check.get_resource_dir(clang_bin, context)
 
         try:
-            with open(args.tidy_args_cfg_file, 'rb') as tidy_cfg:
+            with open(args.tidy_args_cfg_file, 'r', encoding='utf-8',
```

Comment:
I think we don't have test cases for this. It would be good to add a new one to avoid such problems.

Reply:
don't we have the same problem with opening the saargs  file?

Reply:
for the saargs file it is done right:
https://github.com/Ericsson/codechecker/blob/master/analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py#L343-L344